trace/tests/unit/v2/test_client_v2.py

```diff
@@ -60,6 +60,15 @@ def make_api(client_obj):
         self.assertIs(api, api_obj)
         self.assertEqual(clients, [client])
 
+    def test_trace_api_existing(self):
+        """Check that the client caches _trace_api."""
+        client = self._make_one(project=self.project,
+                                credentials=_make_credentials())
+        self.assertIsNone(client._trace_api)
```

Comment:
The point of the '_existing' testcase is to assign a known object (e.g., `mock.sentinel.trace_api`) to `client._trace_api` here, and then assert that `client.trace_api` returns that object, rather than creating a new one.

Reply:
That's what L70 does.